cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/SanityCheck.hs

```diff
@@ -88,3 +95,35 @@ foldBlocksAccumulator _ _ allEvents _ _ =
   filterPoolReap :: LedgerEvent -> Bool
   filterPoolReap (PoolReap _) = True
   filterPoolReap _ = False
+
+
+hprop_asyncRegister_sanity_check :: Property
+hprop_asyncRegister_sanity_check = 
```

Comment:
You didn't include this test in `cardano-testnet-test.hs` so it's never executed.